include/seqan3/io/sequence_file/format_fasta.hpp

```diff
@@ -36,6 +29,8 @@
 #include <seqan3/range/shortcuts.hpp>
 #include <seqan3/range/detail/misc.hpp>
 #include <seqan3/range/view/char_to.hpp>
+#include <seqan3/range/view/istreambuf.hpp>
+#include <seqan3/range/view/interleave.hpp>
```

Comment:
you don't need this now as you do not use it because it is slower. I would delete it for now?